### DISCUSSION TITLE
GitHub Discussions for HAFS

### DISCUSSION THREAD

**Discussion by evankalina** [hafs-community/HAFS#79] [General]
A reminder that as of July 2021, the Developmental Testbed Center (DTC) is offering support to HAFS developers through [GitHub discussions in the hafs-community/HAFS repository](https://github.com/hafs-community/HAFS/discussions). This forum is a place for questions, answers, and discussion about the HAFS application level (e.g., configuration files, scripts, rocoto). In addition, HAFS-related questions about the submodules under hafs-community (e.g., ufs-weather-model, GSI, EMC_post) can also be discussed here. Other developers are welcome and encouraged to answer each other's questions. Discussions will also be monitored by DTC staff.

**Comment by BinLiu-NOAA** [comment]
Obsolete.
